src/format/opamTypes.mli

```diff
@@ -273,21 +284,14 @@ type user_action =
 
 (** Solver universe *)
 type universe = {
+  u_pefuniv  : ((string * string), Opam.Packages.package) Hashtbl.t;
+  u_options  : (string * string list * string list);
+  u_tables   : Pef.Pefcudf.tables;
```

Comment:
u_options is a triple (current switch, available switches, and build options - doc, build, tests) . These are used to then convert pef packages to cudf packages.